src/constants/FileDropzone.js

```diff
@@ -0,0 +1,27 @@
+export const baseStyle = {
+  flex: 1,
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+  padding: "60px",
+  borderWidth: 2,
+  borderRadius: 2,
+  borderColor: "#eeeeee",
+  borderStyle: "dashed",
+  backgroundColor: "#fafafa",
+  color: "#bdbdbd",
+  outline: "none",
+  transition: "border .24s ease-in-out",
+};
+
+export const focusedStyle = {
+  borderColor: "#2196f3",
```

Comment:
Is there a way to extract those hex colors as constants and use only one String in multiple places?

Reply:
Probably